x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/epm/constants.tsx

```diff
@@ -36,6 +37,7 @@ export const AssetIcons: Record<KibanaAssetType, IconType> = {
   'index-pattern': 'indexPatternApp',
   search: 'searchProfilerApp',
   visualization: 'visualizeApp',
+  map: 'mapApp',
```

Comment:
Not sure if these need to be actual names of icons or something? @jen-huang @neptunian @jfsiii

Reply:
I believe this is only used in the AssetsFacetGroup component to get the icon. We have a prototype of an assets section in the package detail page, but its been pushed back for a while as its not really important.  so you can keep this and when we bring it back, we'll fix it if its not working.

Reply:
Ah ok, thanks Sandra!